README.md

```diff
@@ -1,13 +1,30 @@
-# meteodata-lab
```

Comment:
I believe that there should still be a top level header (also for accessibility reasons)